distribution/lib/Standard/Base/0.0.0-dev/src/Data/Statistics.enso

```diff
@@ -279,3 +319,46 @@ Vector.compute self statistic=Statistic.Count =
 Vector.compute_bulk : Vector Statistic -> Vector Any
 Vector.compute_bulk self statistics=[Statistic.Count, Statistic.Sum] =
     Statistic.compute_bulk self statistics
+
+## Compute a single running statistic on the vector.
+
+   Arguments:
+   - statistic: Statistic to calculate.
+Vector.running : Statistic -> Vector Any
+Vector.running self statistic=Statistic.Count =
+    Statistic.running self statistic
+
+## PRIVATE
+compute_fold c v =
```

Comment:
This method is _very_ cryptic. Can we make the parameters a bit more meaningful?
```suggestion
compute_fold acc value =
```

Reply:
I think we should go in a direction of creating a Style Guide for Enso, although probably not something before the release.

Still, we do have style guides for Scala/Java/Rust and at least the first one [discourages usage of single letter variable names unless they are iterator indices](https://enso.org/docs/developer/enso/style-guide/java.html#naming).